bridge/_files.php

```diff
@@ -0,0 +1,115 @@
+<?php
```

Comment:
These files represents the same from the old required dependencies list, including the order.